src/test/java/com/salesforce/storm/spout/sideline/persistence/ZookeeperPersistenceAdapterTest.java

```diff
@@ -338,7 +357,27 @@ public void testEndToEndRequestStatePersistenceWithValidationWithIndependentZkCl
         final String storedDataStr = new String(storedDataBytes, Charsets.UTF_8);
         logger.info("Stored data string {}", storedDataStr);
         assertNotNull("Stored data string should be non-null", storedDataStr);
-        assertEquals("Got unexpected state", expectedStoredState, storedDataStr);
+        Map expectedMap = gson.fromJson(expectedStoredState, HashMap.class);
+
+        System.out.println(expectedStoredState);
```

Comment:
Probably replace these with logger statements?

Reply:
Whoops, good catch.

Reply:
Dropped altogether during the rebase.